.travis.yml

```diff
@@ -27,6 +27,8 @@ virtualenv:
 
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone https://github.com/OCA/account-closing.git ${HOME}/account-closing
+  - git clone https://github.com/OCA/server-tools.git ${HOME}/server-tools
```

Comment:
Those repositories should be added to oca_dependencies.txt and don't need to be added here, I think.